### ISSUE TITLE
Only latin local fonts

### ISSUE THREAD

**Issue by whizz** [janraasch/hugo-scroll#71]
The change introduced #48, while generally a step in the right direction, unfortunately broke my non-latin site, because the local fonts are only in latin set, which does not contain some Czechs letters (such as ě,š,...) and I am sure many others from other languages.

I could submit a PR which would include the full font character set, but that would of course increase the size of the font files.

Alternatively, there could be some configuration setting to select the desired sets, but that will be quite complicated to do and use. 

Or I can do that locally on my site, replacing the fonts.css with my own version and add the proper font files.

What do you think, @janraasch ? BTW: Thank you so much for creating the theme and sharing it with us.

**Comment by janraasch** [comment]
Hi @whizz, thank you for getting in touch 👍.

> Alternatively, there could be some configuration setting to select the desired sets, but that will be quite complicated to do and use.

I agree 💯- `%`. I'd say that's *out of scope*.

**Could you open a pr** 

> which would include the full font character set

**please** 🙏? **Then**, we can check how much of an increase in filesize this is. Maybe it's not that much 😇 ☺️?!

**If** it seems too big, we'll go with

> Or I can do that locally on my site, replacing the fonts.css with my own version and add the proper font files.

Sounds like a plan?

--------

> BTW: Thank you so much for creating the theme and sharing it with us.

Thank you for the kind words. You have to thank [Yonatan Wolowelsky](https://github.com/grmmph) for the original theme though. I just ported it from [Ghost](https://github.com/grmmph/GhostScroll) to Hugo 🤓😄.

**Comment by whizz** [comment]
Apparently, Open Sans Condensed is no longer available at Google Fonts 🤦‍♂️ I'll have to figure out what to do about it.

**Comment by TheLastGimbus** [comment]
Lighthouse gives me this warning:

> Ensure text remains visible during webfont load 
>> Leverage the font-display CSS feature to ensure text is user-visible while webfonts are loading. [Learn more.](https://web.dev/font-display/?utm_source=lighthouse&utm_medium=lr)

Could you add `font-display: swap;` to all fonts? Or can i make a PR? 

But the way, i 2nd this issue. I would like to have easy option to disable custom fonts

**Comment by zjedi** [comment]
@whizz This is also issue for me, you you have any specific fonts in mind?

**Comment by juanMarinero** [comment]
Maybe my quick post helps: [Fonts globally & for Hugo](https://juanmarinero.github.io/blogs/nerd_fonts_and_more/)

For Google Fonts in Hugo Scroll [this section](https://juanmarinero.github.io/blogs/nerd_fonts_and_more/#great-vibes)

One should search and follow previous install steps for each *font supporting wider set of diacritics and accented characters*.

TLDR: my post guides alike your proposal
> locally on my site, replacing the fonts.css with my own version and add the proper font files.
